frontend/src/components/popup/accept_transfer_popup.ts

```diff
@@ -56,6 +56,18 @@ class TransferPopup extends MobxLitElement {
   private handleYes() {
     this.analyticsService.trackButtonClick(ButtonClick.TRANSFER_ACCEPT);
     this.participantService.acceptParticipantTransfer();
+
+    // Move to next stage.
+    const nextStageId = this.experimentService.getNextStageId(
```

Comment:
Can we have a param for moving to next stage? This works well if participants are sitting in a transfer stage, but not if, say, a participant is in a chat stage (but everyone else drops out) and the experimenter transfers them to a different cohort's chat.

Reply:
Actually, a param might be complicated. My understanding is that we previously handled this by having it move to next stage if the participant is currently in a transfer stage, else don't move. Is there something wrong with that logic (i.e., maybe we could just fix/add that)?

Reply:
I don't think we have logic that moves participants to the next stage if they're in a transfer stage currently; I've implemented this here (only move to next stage if on a transfer stage).